lib/ansible/modules/packaging/os/dnf.py

```diff
@@ -287,14 +287,20 @@ def fetch_rpm_from_url(self, spec):
         try:
             rsp, info = fetch_url(self.module, spec)
             if not rsp:
-                self.module.fail_json(msg="Failure downloading %s, %s" % (spec, info['msg']))
+                self.module.fail_json(
+                    msg="Failure downloading %s, %s" % (spec, info['msg']),
+                    results=[]
```

Comment:
Would you add trailing commas, please?

Reply:
Note.... this isn't a dictionary or list definition; it's a function call.  So the normal python style would be to not have a trailing comma....

Reply:
Out of curiosity, I went looking for the reason why the idiom is different for dictionaries and lists vs function calls and found it in PEP8:

> When trailing commas are redundant, they are often helpful when a version control system is used, when a list of values, arguments or imported items is expected to be extended over time.

Function calls are usually not extended arbitrarily later while dicts and lists of hardcoded values often are.  In the case of fail_json(), extra keyword arguments can be treated as more values to put into the results dict so it's not a strict contract but it is unlikely that we'd extend any given function call with more values.  So I'd lean towards not adding a trailing comma here.

Reply:
I normally add commas to not have to think about what it is. Also, when calling functions you don't know whether they'll be extended in future or know. It's one more thing to think about, so I just always automatically do this to multiline function args.